.github/workflows/windows-tests.yml

```
@@ -0,0 +1,54 @@
+name: Windows tests
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
```

The branch should be `main`:
```suggestion
  push:
    branches:
    - main
  pull_request:
    branches:
    - main
```